tutorials/wpa-enterprise.rst

```diff
@@ -0,0 +1,373 @@
+How to Set Up WPA Enterprise (EAP-TTLS-PAP) authentication
+==========================================================
+
+In this tutorial, we will guide you set up WPA Enterprise (EAP-TTLS-PAP)
+authentication for WiFi networks using OpenWISP. The RADIUS capabilities
+of OpenWISP provides integration with FreeRADIUS to allow users to
+authenticate with their django user accounts. Users can either be created
+manually via the admin interface, generated or imported from CSV.
+
+You can try this feature on our :doc:`OpenWISP Demo System <./demo>`!
+
+Pre-requisites for following this tutorial
+------------------------------------------
+
+Enabled OpenWISP RADIUS
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+    You don't need to do anything if you are following this tutorial on
+    the :doc:`OpenWisp Demo System <./demo>`. OpenWISP RADIUS is already
+    enabled on the :doc:`OpenWISP Demo System <./demo>`.
+
+Your OpenWISP installation should have the RADIUS module enabled. If not,
+you can follow the steps at `Enabling the RADIUS module in the OpenWISP
+22.05 ansible role documentation
+<https://github.com/openwisp/ansible-openwisp2/tree/22.05#enabling-the-radius-module>`_.
```

Comment:
The WPA enterprise deployment automation needs the dev version of ansible-openwisp2, it doesn't make sense to link 22.05.